### DISCUSSION TITLE
Problems joining Discord server

### DISCUSSION THREAD

**Discussion by akarelas** [cyprusjs/CyprusJS#39] [Q&A] [answered]
Have you people managed to join the group's Discord server? I've clicked on the link ~10 times, but keep getting the list of my Discord friends, and CyprusJS is not appearing on the left sidebar that lists all the Discord servers I have joined.

But I'm not viewing any error message either.

Have you managed to join?

**Comment by PatrickHeneise** [comment, marked as answer]
Thanks for reporting this and sorry for the inconvenience. I didn't realize the standard invite links have an expiration. Here's a new one that doesn't expire:

https://discord.gg/3xKSyZM4mB
